contrib/qpl-cmake/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set (QPL_SRC_DIR "${ClickHouse_SOURCE_DIR}/contrib/qpl/sources")
 set (QPL_BINARY_DIR "${ClickHouse_BINARY_DIR}/build/contrib/qpl")
 set (UUID_DIR "${ClickHouse_SOURCE_DIR}/contrib/qpl-cmake")
 
-set (EFFICIENT_WAIT ON)
+set (EFFICIENT_WAIT OFF)
```

Comment:
Just curious why the option was turned off? Is it because it is also off in the top-level CMakeLists.txt in the qpl library? Note that we are using `__rdtsc` already in Compression/CompressionCodecDeflateQpl.cpp - so perhaps we can turn it back on?

Reply:
Yes, it is aligned with top-level CMakeLists.txt in the qpl library. And meanwhile the fact is, we don't need EFFICIENT_WAIT any more since we have used "_tpause(1, __rdtsc() + 1000)" already in Compression/CompressionCodecDeflateQpl.cpp.  EFFICIENT_WAIT is the feature inside QPL which use umwait/monitor mechanism  to wait job complete.